src/ui/screens/feed-settings/FeedSettingsScreen.tsx

```diff
@@ -34,6 +35,10 @@ const UNLISTED_EXPLANATION = 'Anyone with a link to your feed can follow it.';
 
 export const FeedSettingsScreen = (props: Props) => {
     const modelHelper = new ReactNativeModelHelper(props.settings.swarmGatewayAddress);
+    const sourceImageUri = modelHelper.getImageUri(props.feed.authorImage);
+    const source = sourceImageUri !== ''
+        ? { uri: sourceImageUri }
+        : defaultImages.userCircle;
```

Comment:
offtopic, coding style 
I've seen that usually you put the semicolon at the end of a ternary ? : operator under the `:`, I prefer it at the beginning of a newline. I see that there is some aesthetics to it, but there are 2 cons in my opinion:
- it's the end of the operation, and usually we put that to the beginning of a newline (curly brace, closing parenthesis, etc.)
- it's also misleading because the `?` and `:` are nicely highlighted otherwise
Also I don't like write these things into stone, I've seen a very different way of writing a multi-level ternary conditional, which had a better readability in that case. 

This got a bit long and I'm not that emotional about it, just wanted to share my thoughts :-)

Reply:
I was experimenting with different styles, we can stick to the semicolon in the beginning of a newline. Unfortunately there is no `tslint` rule for that.